x-pack/plugins/security_solution/common/api/detection_engine/index_management/read_privileges/read_privileges.schema.yaml

```diff
@@ -0,0 +1,43 @@
+openapi: 3.0.0
+info:
+  title: Read privileges API endpoint
+  version: '2023-10-31'
+paths:
+  /api/detection_engine/privileges:
+    get:
+      x-labels: [serverless, ess]
+      operationId: GetPrivileges
+      x-codegen-enabled: true
+      summary: Returns user privileges for the Kibana space
+      description: |
+        Retrieves whether or not the user is authenticated, and the user's Kibana
+        space and index privileges, which determine if the user can create an
+        index (.siem-signals-*) for the Elastic Security alerts generated by
```

Comment:
The `.siem-signals-*` index no longer exists. I'd suggest to remove the outdated description -- we'll have to write proper descriptions for most of the schemas and endpoints.

Reply:
I copied the description from existing API reference documentation to have some minimum information.